docs/api.MD

```
@@ -237,25 +349,33 @@ curl "http://localhost:8000/api/prestiti?draw=1&start=0&length=10&attivo=1&stato
 }
 ```
 
-**Loan States**:
+**Loan States** (`prestiti.stato`, as validated in `PrestitiController`):
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)